### ISSUE TITLE
Should zooming control instructions be included?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#347]
For https://github.com/phetsims/QA/issues/503.
The methods for zooming seem to be: 
1 - keyboard navigation with ctrl +, -
2 - Pinch to zoom with fingers
3 - ctrl + mouse wheel

The zoom controls in the menu of browsers do not work. If I were trying to zoom in on something, that would be what I would try. I think it would be helpful to indicate that zooming is possible in the sim and instructions for how to do so. The keyboard nav dialog box may be a good place for this.

**Comment by KatieWoe** [comment]
If we do add instructions, another control I remembered/rediscovered:
4. ctrl + 0 to undo zoom and set sim to default proportions

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe. @emily-phet @terracoda I am curious what you think about this.  All the standard methods for zoom/pan work as they would in another browser. What specifically doesn't work is the browsers UI for controlling zoom
![image](https://user-images.githubusercontent.com/6396244/82707680-87ea1780-9c4a-11ea-861c-015dc6531b9a.png)

Do we need to let the user know this, or include zoom in the keyboard help dialog?

**Comment by emily-phet** [comment]
@jessegreenberg This is because we're controlling magnification ourself, so the sim only responds to interactions it can directly intercept, correct? So interacting with the browsers menu, the sim doesn't know that's happening and can't respond.

**Comment by jessegreenberg** [comment]
That's correct @emily-phet - and the reasons we cannot use "native" browser zoom are in https://github.com/phetsims/scenery/issues/175#issuecomment-494547681.

**Comment by terracoda** [comment]
@emily-phet and @jessegreenberg, I'm not sure how much room we will have in the Keyboard Shortcuts dialog as accessibility features progress and become more numerous and diverse.

What if the location where we now have as the Keyboard Shortcuts buttons was replaced by a pop-up A11y Options menu, and from that menu the learner would have access to instructions for the different features? Such as:
- Keyboard Shortcuts
- Zoom
- Self-Voicing 

I'm not sure this would work since it makes instructions for features kind of deeply embedded. 
I feel like Keyboard Shortcuts may be more sim specific than the other A11y Features, so it might make sense to keep that one less embedded.

If there is room which there may be in this sim, we could just include a Basic Zooming list, like we have Basic Actions.

**Comment by terracoda** [comment]
In addition, since the Zoom feature will be an option for all sims with or without other a11y features, it might make sense to have a general dialog available fro the PhET Menu, kind similar to the About dialog.

**Comment by jessegreenberg** [comment]
> What if the location where we now have as the Keyboard Shortcuts buttons was replaced by a pop-up A11y Options menu,

I really like this, and think this will be important as we layer on more adaptive features.

For zoom, my opinion is that we shouldn't add anything unless we receive feedback from users that the current input method is confusing. The input for zoom is very conventional for interactive content/games. Google Maps, for example, does not zoom with the browser zoom controls in settings.

**Comment by KatieWoe** [comment]
I would add that, even though instructions for zooming may not be needed, an indication that it is possible may be. Personally, if I wanted to zoom, I would only really try one method before I assumed it couldn't be done at all, and my default would be the browser menu.

**Comment by terracoda** [comment]
@KatieWoe, I agree that it is important to indicate that zooming is possible. The web team has already designed an icon for this feature. The icon will appear on the sim tiles and on the sim pages, so it will be communicated that the sim is indeed zoomable (which is particularly useful for small mobile devices).

The zoom feature will be rolled out generally to several simulations with or without other A11y features. And this will likely happen relatively soon.

We feel the broader team needs to decide how to communicate how this feature should be communicated, in general, on the website and in the teacher tips. 

@emily-phet, @jessegreenberg and I discussed this issue at today's MAL meeting, and feel it is reasonable to mark as won't fix for MAL and open a new issue that opens this discussion up to the broader team.

**Comment by terracoda** [comment]
I'll close this issue since any further discussion will happen in https://github.com/phetsims/website/issues/1064
